qa/build-standalone-example-plugins/build.gradle

```diff
@@ -0,0 +1,58 @@
+import org.elasticsearch.gradle.VersionProperties
+
+import java.util.regex.Matcher
+
+apply plugin: 'elasticsearch.build'
+
+dependencies {
+    testCompile 'commons-io:commons-io:2.6'
+    testCompile gradleTestKit()
+    testCompile (project.project(':test:framework')) {
+        exclude group: "commons-logging"
+    }
+}
+
+task integTest(type: Test) {
+    // integration test requires the local testing repo to simulate unpublished dependencies
+    dependsOn project.rootProject.allprojects.collect {
+        it.tasks.matching { it.name == 'publishNebulaPublicationToTestRepository'}
+    }
+
+    exclude "**/*Tests.class"
+
+    FileCollection examplesToTest = files(
+            project(':example-plugins').subprojects.collect { it.projectDir }
+    )
+    inputs.files(examplesToTest.sort())
```

Comment:
File order doesn't matter here (only for classpaths). We should declare relative path sensitivity though.